PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,12 @@
 - [ ] Enhancement (improves an existing feature and functionality)
 - [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
 
+<!-- If this PR is to fix an issue on GH, uncomment the section and provide the id of issue -->
+<!-- If "Fixes: #<id> is specified, the issue will automatically be closed when PR gets merged -->
+
```

Comment:
That's true, it can (and should) even be in the commit description as well. I only added another section since we already have some other sections for different purposes, and make it more unified (in terms of placement of `Fixes: #<id>`)

Reply:
That's true! Although the correct syntax would be `Fixes #..., Fixes #..., ...`. The keyword needs to be specified before number everytime ([reference](https://help.github.com/articles/closing-issues-using-keywords/#closing-multiple-issues)).